.github/workflows/build-and-deploy.yml

```diff
@@ -45,6 +45,13 @@ jobs:
           node renamer.js ../devonfw-guide/ Home .asciidoc
           cd ..
 
+      - name: Solutions from Stack repositories
+        run: |
+          cd architectures/scripts
+          npm install 
+          node copySolutionsFromStack.js ../../devonfw-guide
+          cd ..
```

Comment:
You need to go up two levels because to are going down two levels in line 50. So please change this to "cd ../../"